ci/recipe/meta.yaml

```diff
@@ -1,4 +1,4 @@
-{% set version = '0.2018.2' %}
+{% set version = '2018.4.0' %}
```

Comment:
BTW - 2018.6 is [scheduled](https://forum.qiime2.org/t/preview-qiime-2-2018-6-development-changelog/4037/12) to come out this week.

Reply:
good to know. I am not sure if I can manage to add in all little fixes by that time, but I will try and raise to 2018.6 then.